context/build-utils/webpack.common.js

```diff
@@ -77,7 +77,13 @@ const config = {
       },
     ],
   },
-  plugins: [new CleanWebpackPlugin()],
+  plugins: [
+    new CleanWebpackPlugin(),
+    new webpack.DefinePlugin({
+      // update globals in eslintrc to fix undefined errors
+      IMAGES_CDN_ORIGIN: JSON.stringify('https://d3evp8qu4tjncp.cloudfront.net'),
```

Comment:
I had been thinking you would pass this in with the other environment variables... I guess I have a slight preference for minimizing the range of techniques we use, so if someone is familiar with the other configurations in our codebase, they'll have the right intuition about how to work with this one.

(If you do decide to change it, I don't think it needs to be exactly the same, because for now while if would come from Flask, it wouldn't need to originate with an envvar Zhao would need to add.)

Also:
```suggestion
      CDN_URL: JSON.stringify('https://d3evp8qu4tjncp.cloudfront.net'),
```